company-profiles/epsy.md

```diff
@@ -0,0 +1,35 @@
+# Epsy Health
```

Comment:
Is the company called `Epsy` or `Epsy Health`. Please rename this or the other aspects of the PR to make everything consistent.